_includes/org-table.html

```diff
@@ -16,7 +16,7 @@ <h6 class="no-matches" style="display: none;">No matches.</h6>
         <tr>
           <td>
             <a href="https://github.com/{{ org }}" title="{{ org }}">
-            <img src="https://github.com/{{ org }}.png" width="40" height="40" alt="{{ org }}"></a>
+            <img src="https://avatars.githubusercontent.com/{{ org }}?v=3&amp;s=40" width="40" height="40" alt="{{ org }}"></a>
```

Comment:
This should just be `avatars.githubusercontent.com`. It'll work, but you want to keep the host names consistent across your site so browsers and CDNs aren't storing multiple copies in their caches.

On a dynamic site, I'd suggest using consistent hashing on the path (`/org`) to get a consistent modulus for the subdomain (`avatars0`-`avatars3`). I don't think we can do that here without some kind of custom plugin, so I'd recommend just using `avatars`.